sim/lib/settings/town_settings_sanfrancisco.py

```diff
@@ -83,47 +83,31 @@ def _essential_distribution():
     0])     # 80+
     return ed
 
-
-def _essential_mobility(wtype):
-    # site types: ['education', 'office', 'social', 'supermarket']
-    if wtype in ['education', 0]:
-        essential_mob_rate_per_type = [5.0, 0.0, 3.6, 0.27]
-        essential_dur_mean_per_type = [5.0, 0.1, 0.64, 0.4]
-        essential_variety_per_type = [1, 1, 10, 2]
-    elif wtype in ['office',1]:
-        essential_mob_rate_per_type = [0.0, 5.0, 3.6, 0.27]
-        essential_dur_mean_per_type = [0.1, 5.0, 0.64, 0.4]
-        essential_variety_per_type = [1, 1, 10, 2]
-    elif wtype in ['social',2]:
-        essential_mob_rate_per_type = [0.0, 0.0, 5.0, 0.27]
-        essential_dur_mean_per_type = [0.1, 0.1, 5.0, 0.4]
-        essential_variety_per_type = [1, 1, 10, 2]
-    elif wtype in ['supermarket',3]:
-        essential_mob_rate_per_type = [0.0, 0.0, 3.6,  5.0]
-        essential_dur_mean_per_type = [0.1, 0.1, 0.64, 5.0]
-        essential_variety_per_type = [1, 1, 10, 2]
-    else:
-        return None, None, None
-    return essential_mob_rate_per_type, essential_dur_mean_per_type, essential_variety_per_type
-
 def _worker_mobility():
     worker_mob_rate_per_types = [
-        [5.0, 0.0, 3.6, 0.27, 0.5],
-        [0.0, 5.0, 3.6, 0.27, 0.5],
-        [0.0, 0.0, 5.0, 0.27, 0.5],
-        [0.0, 0.0, 3.6,  5.0, 0.5]
+        [5.0, 0.0, 1.88, 3.6, 0.27, 0.5],
+        [0.0, 5.0, 1.88, 3.6, 0.27, 0.5],
+        [0.0, 0.0, 5.0,  3.6, 0.27, 0.5],
+        [0.0, 0.0, 1.88, 5.0, 0.27, 0.5],
+        [0.0, 0.0, 1.88, 3.6,  5.0, 0.5],
+        [0.0, 0.0, 1.88, 3.6, 0.27, 5.0] # placeholder. We don't have workers in home gatherings
     ]
     worker_dur_mean_per_types = [
-        [5.0, 0.1, 0.64, 0.4, 3.0],
-        [0.1, 5.0, 0.64, 0.4, 3.0],
-        [0.1, 0.1, 5.0, 0.4, 3.0],
-        [0.1, 0.1, 0.64, 5.0, 3.0]
+        [5.0, 0.1, 0.55, 0.64, 0.4, 3.0],
+        [0.1, 5.0, 0.55, 0.64, 0.4, 3.0],
+        [0.1, 0.1, 5.0,  0.64, 0.5, 3.0],
+        [0.1, 0.1, 0.55,  5.0, 0.4, 3.0],
+        [0.1, 0.1, 0.55, 0.64, 5.0, 3.0],
+        [0.1, 0.1, 0.55, 0.64, 0.4, 5.0] # placeholder. We don't have workers in home gatherings
+        
     ]
-    worker_variety_per_types = [
-        [1, 1, 10, 2, 1],
-        [1, 1, 10, 2, 1],
-        [1, 1, 10, 2, 1],
-        [1, 1, 10, 2, 1]
+    worker_variety_per_types = [ # this is not used in simulations
```

Comment:
This is actually not used in the code, but I still keep it here so that I don't need to modify more things...

Reply:
This array IS still used.  So we should make sure it has a sensible number for retail stores, 10 seems ok to me.  It is used to determine the site variety for site types other than your assigned worksite.  See: https://github.com/stanfordfuturebay/simulator/blob/4f79d722235e8770a384798fd0839abd917cd6cf/sim/lib/mobilitysim.py#L176

Reply:
This variety_per_type is defined in the town_generator and is the same for everyone. worker_variety_per_types is not included into kwargs.

Reply:
I see now, you are correct, and you added the retail variety to `variety_per_type`, so this seem good.